src/bin/main.rs

```diff
@@ -126,6 +126,12 @@ fn make_opts() -> Options {
          `current` writes to stdout current config as if formatting the file at PATH.",
         "[default|minimal|current] PATH",
     );
+    opts.optflag(
+        "l",
+        "files-with-matches",
```

Comment:
IMHO we should use a slightly more rustfmt-oriented name for this. I would rather change this to something like `files-with-diff`.

Reply:
Will do!

Reply:
Done!